esphomeyaml/components/sensor/total_daily_energy.rst

```diff
@@ -40,6 +41,7 @@ Configuration variables:
 - **name** (**Required**, string): The name of the sensor.
 - **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
 - All other options from :ref:`Sensor <config-sensor>` and :ref:`MQTT Component <config-mqtt-component>`.
+- **time_id** (**Required**, :ref:`config-id`): The ID of the time component.
```

Comment:
a) Not required
b) I don't like exposing these internal configuration options (for example also `make_id` and `mqtt_id`) as the user doesn't need to know about these. It's auto-detected as long as you have a valid time platform in your config.

Reply:
agree. I did not have tzlocal installed so it complained about missing time component. tried to resolve by adding the time_id. Installing tzlocal fixed all. Thanks a lot! I really like your software.